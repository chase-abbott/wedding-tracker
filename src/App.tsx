import React from 'react';
import Heading from './components/Heading';
import TimerContainer from './containers/TimerContainer';
import './App.css';

function App() {
  return (
    <div className="flex flex-col items-center justify-center">
      <Heading/>
      <TimerContainer/>
    </div>
  );
}

export default App;
