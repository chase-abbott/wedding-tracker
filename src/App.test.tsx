import React from 'react';
import { render, cleanup } from '@testing-library/react';
import App from './App';

// global.fetch = jest.fn(() => Promise.resolve({
//   json: () => {
//     return (
//       [{
//         EpochTime: 1234,
//         HasPrecipitation: true,
//         IsDayTime: true,
//         Link: 'string',
//         LocalObservationDateTime: 'string',
//         MobileLink: 'string',
//         PrecipitationType: 'string',
//         Temperature: { Imperial: { Value: 60 }, Metric: { Value: 10 } },
//         WeatherIcon: 12,
//         WeatherText: 'string'
//       }]
//     );
//   }
// }));
jest.useFakeTimers();
describe('renders app', () => {
  afterAll(() => cleanup());


  it('renders learn react link', () => {
    const { asFragment } = render(<App />);
    
    expect(asFragment()).toMatchSnapshot();
  });
});

