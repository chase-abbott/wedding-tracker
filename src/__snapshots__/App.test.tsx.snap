// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders learn react link 1`] = `
<DocumentFragment>
  <div
    class="flex flex-col items-center justify-center"
  >
    <h1
      class="text-pink-200 text-4xl tablet:text-6xl laptop:text-7xl font-sans font-bold m-4"
    >
      Wedding Tracker
    </h1>
    <h1>
       Timer 
    </h1>
  </div>
</DocumentFragment>
`;
